docs/content/preview/yugabyte-cloud/cloud-admin/_index.md

```diff
@@ -41,7 +41,7 @@ You must be signed in as an Admin user to perform Admin tasks. Developer users c
         <div class="title">Manage billing</div>
       </div>
       <div class="body">
-        Manage your billing profile and payment methods.
+        Manage your billing profile and payment methods, and view your running costs and past invoices.
```

Comment:
Suggestion:

Manage your billing profile and payment methods, as well as view your running costs and past invoices.